prdoc/pr_6838.prdoc

```diff
@@ -0,0 +1,9 @@
+title: 'Snowbridge: Ensure source always from Assethub for exported message'
+doc:
+- audience: Runtime Dev
+  description: |-
+    Ensure the last hop of exported message is always from AH. 
+    So user can't export message from non system parachain directly.
+crates:
+- name: snowbridge-router-primitives
+  bump: patch
```

Comment:
```suggestion
- name: snowbridge-router-primitives
  bump: minor
- name: testnet-parachains-constants
  bump: patch
- name: bridge-hub-westend-runtime
  bump: patch
```

The first one is minor since you add an additional export, the two other ones it's important to mention or they won't get bumped. It's particularly important to mention bridge-hub-westend-runtime to test changes on westend.